packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-execution.js

```diff
@@ -489,3 +521,24 @@ test.serial('writeExecution() correctly sets both original_payload and final_pay
   t.deepEqual(pgRecord.original_payload, originalPayload);
   t.deepEqual(pgRecord.final_payload, finalPayload);
 });
+
+test.serial('writeExecution() calls publishExecutionSnsMessage and successfully publishes an SNS message', async (t) => {
+  const {
+    cumulusMessage,
+    executionArn,
+    knex,
+    QueueUrl,
+  } = t.context;
+
+  await writeExecution({ cumulusMessage, knex });
+
+  const { Messages } = await sqs().receiveMessage({ QueueUrl, WaitTimeSeconds: 10 }).promise();
+
+  t.is(Messages.length, 1);
+
+  const snsMessage = JSON.parse(Messages[0].Body);
+  const executionRecord = JSON.parse(snsMessage.Message);
+
+  t.deepEqual(executionRecord.arn, executionArn);
+  t.deepEqual(executionRecord.status, cumulusMessage.meta.status);
```

Comment:
You should use `t.is` when comparing specific properties unless the value of those properties are objects/arrays. 

But really I think we should do a `deepEqual` compare between the `executionRecord` and a translation of PG -> API of the record in PG, which will verify that we are sending over the exact expected record